internal/handlers/handler_test.go

```diff
@@ -558,6 +571,26 @@ func TestReadOnlyHandlers(t *testing.T) {
 			),
 		},
 
+		"HostInfo": {
+			req: types.MustMakeDocument(
+				"hostInfo", int32(1),
+			),
+			resp: types.MustMakeDocument(
+				"system", types.MustMakeDocument(
+					"currentTime", fmt.Sprintf("ISODate(%s)", nowStr),
+					"hostname", hostname,
+					"cpuAddrSize", fmt.Sprintf("%d", strconv.IntSize),
+					"numCores", fmt.Sprintf("%d", runtime.NumCPU()),
+					"cpuArch", runtime.GOARCH,
+					"numaEnabled", "false",
```

Comment:
I think that should be bool, not string. Again, diff output in `make run` should show that

Reply:
Yep, fixes (as well as other fields). Thanks again.